Package/Installer.cs

```diff
@@ -405,7 +405,6 @@ enum AbortOrRetryResponse
     }
 
     [Obfuscation(Exclude = true)]
-    [Display("Package files are in use")]
     class AbortOrRetryRequest
```

Comment:
Now you removed this, but does that not affect the original use of the code?

Reply:
You're right, I forgot to add a generic title :)